docs/lib/storage/fragments/ios/getting-started.md

```diff
@@ -98,7 +98,6 @@ Add the following imports to the top of your `AppDelegate.swift` file:
 
 ```swift
 import Amplify
-import AWSMobileClient
 import AmplifyPlugins
 ```
 
```

Comment:
looks like this entire file can be removed now that it is using the `getting-started/common.md`.

Reply:
Yeah, I have removed this getting-started from Analytics, Haven't removed here. Going to remove it